### DISCUSSION TITLE
How to setup Playnite plugin

### DISCUSSION THREAD

**Discussion by slashalbaraa** [Flow-Launcher/Flow.Launcher#1327] [Q&A]
(playnite plugin) I installed the app and extension and flowlauncher plugin my issue is "Library not found" ive placed the directory (D:/Playnite) but its still not found, i think its the wrong directory so what's right?

**Comment by Garulf** [comment]
It _should_ in most cases automatically find your Playnite directory. 

But if for whatever reason it does not you may enter:

```
C:/Users/%USERNAME%/AppData/Roaming/Playnite
```

Replacing your `%USERNAME%` with your user folder name.
